public/editor/scripts/editor/js/fc/publisher.js

```diff
@@ -186,6 +186,13 @@ define(function(require) {
     if (publisher.unpublishing) {
       return;
     }
+
+    var didConfirm = window.confirm("Are you sure? The current link will stop working and if you publish again, the link will be different.");
```

Comment:
Ah, nice catch! @O-kasso, you can find information about how to localize this here: https://github.com/mozilla/thimble.mozilla.org/wiki/Localization#how-it-works and https://github.com/mozilla/thimble.mozilla.org/wiki/Localization#ui-strings